go/kubebuilder/memcached-operator/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-This Memcached operator is a simple example operator using the [Operator SDK][operator_sdk] CLI tool and controller-runtime library API.
+This Memcached operator is a simple example operator using the [Operator SDK][operator_sdk] CLI tool and controller-runtime library API, using validation webhooks.
```

Comment:
Since webhooks are a typical part of an operator, you don't need to explicitly say that it uses them.
```suggestion
This Memcached operator is a simple example operator using the [Operator SDK][operator_sdk] CLI tool and controller-runtime library API.
```